system/helpers/roundHelpers.mjs

```diff
@@ -0,0 +1,87 @@
+/**
+ * Formats rounds for human readable display
+ *
+ * 1 round is nominally a "turn"
+ * 10 rounds is a minute
+ * 600 rounds is an hour
+ * 14400 rounds is a day
+ * 1008000 is a week
+ * 4032000 is a month
+ * 48384000 is a year
+ *
+ * @param {number} rounds
+ * @returns {{number: number, unit: string}}
+ */
+export function formatRounds(rounds) {
+  if (rounds < 10) {
+    return {
+      number: rounds,
+      unit: "turns"
```

Comment:
We'll want to localize these, probably with a `label` on top of a machine-friendly `unit` that we already have